ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/calico/hypershift/private/guest/cucushift-installer-rehearse-aws-ipi-calico-hypershift-private-guest-workflow.metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/calico/hypershift/private/guest/cucushift-installer-rehearse-aws-ipi-calico-hypershift-private-guest-workflow.yaml",
```

Comment:
How about moving `calico` to `aws/ipi/ovn/hypershift/private/` folder ?

Reply:
I think calico should be at the same level as "cilium" and "ovn" because it's another CNI implementation, similar to those two.
We now have:
* aws/ipi/cilium
* aws/ipi/ovn
* aws/ipi/calico

Reply:
Yeah, actually if possible, I even advice move `aws/ipi/cilium` together.